### DISCUSSION TITLE
Remove the delay when pressing Escape during gameplay

### DISCUSSION THREAD

**Discussion by Mechanizen** [ppy/osu#29571] [Ideas] [answered]
As someone used to press Escape and click retry during gameplay, it is very frustrating to have that hold time before the Escape menu pops up. Lazer UI does a great job at doing flawless transitions between different interfaces, I don't see why this particular feature should have a wait time. 

The only reason for this delay to be there would be sort of a "security" but I really can't think of a situation where the user would press Escape by misstake during gameplay. Maybe it was to make it feel the same as the retry button? I don't know.

In any case, I wish this delay was removed or at least I wish there was an option to disable it because it makes retrying very slow and frustrating compared to stable.

**Comment by Joehuu** [comment, marked as answer]
This setting is what you're looking for:

![osu!_TAs6YgzjBp](https://github.com/user-attachments/assets/f815889e-c5e3-465a-9d2a-0260ce3af469)

**Reply by Mechanizen** [reply]
Thanks, I tried to search for an "Escape" option thing but I couldn't figure it out
